docs/docs/development/documentation.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Documentation
+sidebar_label: Documentation
+---
+
+:::note
+The documentation is written with [docusaurus](https://docusaurus.io/). You don't need install anything specific for docusaurus but it can be helpful to reference their documentation if needed.
```

Comment:
"Docusaurus" is a proper noun and should be capitalized.

The bit about not needing to install anything might be confusing, because you do need Node if not going through Docker. Maybe either clarify that the Docker image has everything you need or just drop the part of this sentence about installing things?